pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/Constants.java

```diff
@@ -33,4 +33,5 @@ private Constants() {
   public static final String THETA_TUPLE_SKETCH_SAMPLING_PROBABILITY = "samplingProbability";
   public static final String PERCENTILETDIGEST_COMPRESSION_FACTOR_KEY = "compressionFactor";
   public static final String SUMPRECISION_PRECISION_KEY = "precision";
+  public static final String KLL_DOUBLE_SKETCH_K = "K";
```

Comment:
Do we want to make this capital `K`? Also calling it `DOUBLE_SKETCH_K` is a little bit confusing because it is actually an `int`.
If we follow the same format as `HLL`, it should be `KLL_SKETCH_K_KEY = "k"`

Reply:
I have tested this by creating a dummy table and ingesting sketch data to trigger minion jobs. The merge is working as expected. I have added details in description. 

`K` was chosen to make it consistent with convention on apache docs. Let me know if we need to change this. 

https://datasketches.apache.org/docs/KLL/KLLSketch.html

Reply:
+1, I think we should stick to `K`. Also I think the naming is because we're using the `KllDoublesSketch` implementation of `KllSketch`